client/src/elm/Pages/NCD/RecurrentActivity/Model.elm

```diff
@@ -0,0 +1,111 @@
+    | SetCreatinineResult String
+    | SetUreaResult String
+    | SetNitorogenResult String
+    | SaveCreatinineResult PersonId (Maybe ( NCDCreatinineTestId, NCDCreatinineTest )) (Maybe LaboratoryTask)
+    | SetAltResult String
+    | SetAstResult String
+    | SaveLiverFunctionResult PersonId (Maybe ( NCDLiverFunctionTestId, NCDLiverFunctionTest )) (Maybe LaboratoryTask)
+      -- NextStepsMsgs
+      -- | SetActiveNextStepsTask NextStepsTask
+      -- | SetReferralBoolInput (Bool -> ReferralForm -> ReferralForm) Bool
+      -- | SetFacilityNonReferralReason (Maybe ReasonForNonReferral) ReferralFacility ReasonForNonReferral
+      -- | SaveSendToHC PersonId (Maybe ( NCDSendToHCId, NCDSendToHC )) (Maybe NextStepsTask)
+      -- | SetMedicationDistributionBoolInput (Bool -> MedicationDistributionForm -> MedicationDistributionForm) Bool
+      -- | SetMedicationDistributionAdministrationNote (Maybe AdministrationNote) MedicationDistributionSign AdministrationNote
+      -- | SetRecommendedTreatmentSign (List RecommendedTreatmentSign) RecommendedTreatmentSign
+      -- | SaveMedicationDistribution PersonId (Maybe ( NCDMedicationDistributionId, NCDMedicationDistribution )) (Maybe NextStepsTask)
+      -- | SetHealthEducationBoolInput (Bool -> HealthEducationForm -> HealthEducationForm) Bool
+      -- | SaveHealthEducation PersonId (Maybe ( NCDHealthEducationId, NCDHealthEducation )) (Maybe NextStepsTask)
+    | CloseLabsResultsEntry PersonId NCDLabsResultsId LabsResultsValue
+
+
+type alias Model =
+    { labResultsData : LabResultsData
+
+    -- , nextStepsData : NextStepsData
+    }
+
+
+emptyModel : Model
+emptyModel =
+    { labResultsData = emptyLabResultsData
+
+    -- , nextStepsData = emptyNextStepsData
+    }
+
+
+type alias LabResultsData =
+    { randomBloodSugarTestForm : RandomBloodSugarResultForm NCDEncounterId
+    , urineDipstickTestForm : UrineDipstickResultForm
+    , creatinineTestForm : CreatinineResultForm
+    , liverFunctionTestForm : LiverFunctionResultForm
+    , activeTask : Maybe LaboratoryTask
+    }
+
+
+emptyLabResultsData : LabResultsData
+emptyLabResultsData =
+    { randomBloodSugarTestForm = emptyRandomBloodSugarResultForm
+    , urineDipstickTestForm = emptyUrineDipstickResultForm
+    , creatinineTestForm = emptyCreatinineResultForm
+    , liverFunctionTestForm = emptyLiverFunctionResultForm
+    , activeTask = Nothing
+    }
+
+
+
+-- type alias NextStepsData =
+--     { referralForm : ReferralForm
+--     , medicationDistributionForm : MedicationDistributionForm
+--     , healthEducationForm : HealthEducationForm
+--     , activeTask : Maybe NextStepsTask
+--     }
+--
+--
+-- emptyNextStepsData : NextStepsData
+-- emptyNextStepsData =
+--     { referralForm = emptyReferralForm
+--     , medicationDistributionForm = emptyMedicationDistributionForm
+--     , healthEducationForm = emptyHealthEducationForm
+--     , activeTask = Nothing
+--     }
```

Comment:
same here

Reply:
Next steps related development is done at https://github.com/TIP-Global-Health/eheza-app/pull/456 and https://github.com/TIP-Global-Health/eheza-app/pull/459, All commented out code is handled there.